icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -6014,7 +6014,7 @@ class TestCase {
                 new TestCase(MeasureUnit.MILLIGRAM_OFGLUCOSE_PER_DECILITER, "2 milligrams per deciliter"),
                 new TestCase(MeasureUnit.forIdentifier("millimeter-ofhg"), "2 millimeters of mercury"),
                 new TestCase(MeasureUnit.forIdentifier("liter-per-100-kilometer"), "2 liters per 100 kilometers"),
-                new TestCase(MeasureUnit.forIdentifier("permillion"), "2 parts per million"),
+                new TestCase(MeasureUnit.forIdentifier("part-per-1e6"), "2 parts per million"),
```

Comment:
Before ICU 78 goes out, we should really make unit aliases work, so that we don't break callers again as with the metric-ton change.
@younies @sffc please...

Reply:
https://github.com/unicode-org/icu/pull/3670